### PR TITLE
clean up.

### PR DESCRIPTION
**Pull request by chinsyo**

### CODE REVIEW

File: Sources/Vapor/Provider/ProviderInstall.swift [hunk 1]
```diff
@@ -51,19 +51,19 @@ public final class ProviderInstall: Command {
                 console.print("Copied public files")
                 dirty = true
             } catch {
-                //
+                console.print("Copy public files failed")
```
Comment (Joannis) [inline Sources/Vapor/Provider/ProviderInstall.swift]: I'm not sure what's ideal here. But the `console.print` operations should be reviewed by @tanner0101 . The rest looks good to me 👍

Comment (chinsyo) [reply Sources/Vapor/Provider/ProviderInstall.swift]: Thanks for your review.🤝

Comment (tanner0101) [reply Sources/Vapor/Provider/ProviderInstall.swift]: this should be `.warning`

Comment (chinsyo) [reply Sources/Vapor/Provider/ProviderInstall.swift]: @tanner0101 Fixed. Request for review.

**Comment by chinsyo** [comment]
@tanner0101 Request for review.

**Comment by tanner0101** [comment]
thanks :)

**Pull request merged**